mflowgen/backends/make_backend.py

```diff
@@ -141,6 +143,41 @@ def gen_step_directory( s, dst, src, deps, extra_deps, sandbox ):
     s.w.newline()
 
     return [ target ]
+  
+  # gen_subgraph_dir
+  #
+  # Expected semantics
+  #
+  # - Create the {dst}
+  # - Configure {dst} as a build dir for {src}
+  # - Parameterize using the saved YAML in the metadata directory
+  # - This rule depends on {deps}
+  # - {sandbox} does nothing here
```

Comment:
Maybe sandbox doesn't need to be passed then?

Reply:
I left it in to keep the same interface as the gen_step_directory function that it's based on, but I'm also ok with removing it if you'd rather.